authui/src/otpInput.ts

```diff
@@ -1,4 +1,5 @@
 import { Controller } from "@hotwired/stimulus";
+import { TextFieldController } from './authflowv2/text-field';
```

Comment:
Can we apply `text-field` controller in `__otp_input.html`?

Reply:
The issue is the hidden input is not the one with error state. Maybe I add an input container target.